DOCUMENTATION.md

```diff
@@ -99,15 +99,50 @@ This document provides detailed information about each service module and its re
       * [Get Keeper Info](#get-keeper-info)
       * [Get Type And Version](#get-type-and-version-2)
       * [Get Upkeep Transcoder Version](#get-upkeep-transcoder-version)
+  * [Functions Services](#functions-services)
+    * [Service alias: `functionsRouter`](#service-alias-functionsrouter)
```

Comment:
The service is just `Functions`. Functions Router is a specific contract that does subscription management, request handlinge tc.

Reply:
The same as we discussed in your previous comment. Service names in the plugin should be edited for clarity.